README.md

```diff
@@ -492,6 +495,92 @@ status:
     status: 'True'
 ```
 
+## Custom Binding Paths
+
+There are scenarios where the application resource is not PodSpec-able and yet bind `volume`, `volumeMount`, and `env` at custom locations.  The reconciler can also bind the optional `containers` list specified by the user in the `ServiceBinding` resource into the application resource.  So that, the application reconciler can bind containers in the destination Pod based on the `containers` list.  This extension provides the customized binding functionality through a `BindingPathMap` resource.  The `ServiceBinding` reconciler **MUST** watch for the `BindingPathMap` and the `ServiceBinding` resources and react.  The `BindingPathMap` resource **MUST** be a cluster-scoped resource.  The allowed custom binding paths are `volume`, `volumeMount`, `env`, and `containers`.  The value for each of these custom binding paths **MUST** be a string type.  If `volumeMount` is specified, `volume` also **MUST** be defined.  The values of these items **MUST** be dot-delimited paths pointing to the fields on the application resource.  The reconciler **MUST** bind each of these specified values in application resource at the given location.  An exemplar CRD can be found [here][cbp-crd].
+
+1. The `volume` field in the application resource **MUST** conform to the [Volume][volume] API.
+2. The `volumeMount` field in the application resource **MUST** conform to the [VolumeMount][volumemount] API.
+3. The `env` field in the application resource **MUST** conform to the [EnvVar][env] API.
+4. The `containers` field in the application resource **MUST** conform to the API schema given here:
+
+    ```yaml
+    schema:
```

Comment:
Why is this schema relevant? It's neither the schema of `[]corev1.Container` (which is what I'd expect) or matching the schema for the BindingPathMap resource as the examples show.